lib/Standalone/ToBlockLayoutAndBack.cpp

```diff
@@ -417,138 +433,161 @@ struct BlockConv2DNchwFchwLayout
 };
 
 //===----------------------------------------------------------------------===//
-// SinkBlockLayout
+// PropagateThrElementWiseOp
```

Comment:
Convoluted name, no comments. Would be nice to have a comment like matmul above.